src/js_realm.hpp

```diff
@@ -309,6 +311,7 @@ class RealmClass : public ClassDefinition<T, SharedRealm, ObservableClass<T>> {
         {"deleteFile", wrap<delete_file>},
         {"exists", wrap<realm_file_exists>},
         {"_createUserAgentDescription", wrap<create_user_agent_description>},
+        {"_bsonParseJsonForTest", wrap<bson_parse_json>},
```

Comment:
Any reason we couldn't use the [bson](https://www.npmjs.com/package/bson) package for that?